cloud/jenkins/psmdb_operator_aks_latest.groovy

```diff
@@ -12,8 +12,8 @@ void CreateCluster(String CLUSTER_SUFFIX) {
                 az login --service-principal -u "$AZURE_CLIENT_ID" -p "$AZURE_CLIENT_SECRET" -t "$AZURE_TENANT_ID"  --allow-no-subscriptions
                 az account show --query "{subscriptionId:id, tenantId:tenantId}"
                 az account list --all --output table
-                az aks create -g percona-operators -n $CLUSTER_NAME-${CLUSTER_SUFFIX} --load-balancer-sku basic --enable-managed-identity --node-count 3 --node-vm-size Standard_B4ms --min-count 3 --max-count 3 --node-osdisk-size 30 --network-plugin kubenet  --generate-ssh-keys --enable-cluster-autoscaler --outbound-type loadbalancer --kubernetes-version ${params.PLATFORM_VER} -l westeurope
-                az aks get-credentials --subscription Pay-As-You-Go --resource-group percona-operators --name $CLUSTER_NAME-${CLUSTER_SUFFIX} 
+                az aks create -g percona-operators --subscription eng-cloud-dev  -n $CLUSTER_NAME-${CLUSTER_SUFFIX} --load-balancer-sku basic --enable-managed-identity --node-count 3 --node-vm-size Standard_B4ms --min-count 3 --max-count 3 --node-osdisk-size 30 --network-plugin kubenet  --generate-ssh-keys --enable-cluster-autoscaler --outbound-type loadbalancer --kubernetes-version ${params.PLATFORM_VER} -l westeurope
```

Comment:
Should we use spot instances for running these tests?
https://medium.com/@ershivamgupta/using-spot-instances-with-aks-azure-kubernetes-service-node-pool-c26a17c79470

Reply:
It's nice idea) I try to do it again, but for now we can't increase spot instances quota  :( Let's discuss it with Slava after his vacation) 
```
QMS Update - Status: ResourceType: crpCores
 {
	Quota Bucket: TotalLowPriorityCores
	Status Description: Due to very high rates of Spot consumption, Microsoft is unable to approve additional quota at this time
	State: SpotVMNotAllowedForPayGCustomer
	Current Quota: 3
	New Quota: 160
}
Properties: [location, eastus]
}
```